spk/sudo/Makefile

```diff
@@ -0,0 +1,25 @@
+SPK_NAME = sudo
+SPK_VERS = 1.0
```

Comment:
Should be the same value as `cross/sudo`.
`SPK_VERS` is used for the version of the software you're packaging. The package version is `SPK_REV`, and that should be `1` (and go up with every new package release)